src/tools/icon-utils.ts

```diff
@@ -16,16 +16,6 @@ export interface IconPacks {
 export async function loadIconData(plugin: OmnisearchPlugin): Promise<any> {
   const app = plugin.app
 
-  // Check if the 'obsidian-icon-folder' plugin is installed and enabled
-  // Casting 'app' to 'any' here to avoid TypeScript errors since 'plugins' might not be defined on 'App'
-  const iconFolderPlugin = (app as any).plugins.getPlugin(
-    'obsidian-icon-folder'
-  )
-  if (!iconFolderPlugin) {
-    warnDebug('Icon Folder plugin is not installed or not enabled.')
-    return {}
-  }
```

Comment:
You misunderstood :p

The _checks_ are necessary, as you need to verify that 1) the plugin is active and 2) (where applicable) the data.json exists and can be read. What's unnecessary is the call to `warnDebug()` when the plugin isn't active

Reply:
Ahh I get you. You just want to return null if the plugin isnt enabled but surpress the alert.

Sorry, that should have been obvious. My bad.